### PR TITLE
update the main

### PR DESCRIPTION
**Pull request by OmarAbdElNasser260**

### CODE REVIEW

File: src/Main2.java [hunk 1]
```diff
@@ -5,6 +5,9 @@ public class Main2{
  public static void main(String[] args){
  for(int i = 0 ; i < 10; ++i)
    System.out.println("Configuration Management Second Day"); 
+System.out.println("Configuration Management Second Day1"); 
+System.out.println("Configuration Management Second Day2"); 
+System.out.println("Configuration Management Second Day3");
```
Comment (Omniasalman) [inline src/Main2.java:10]: code double

Comment (BasantMohamed33) [inline src/Main2.java:10]: code duplication

Comment (Salwa1012) [inline src/Main2.java:10]: duplicate

Comment (MaryamMagdy570) [inline src/Main2.java:10]: maryam comment

File: src/Main2.java [hunk 2]
```diff
@@ -5,6 +5,9 @@ public class Main2{
  public static void main(String[] args){
  for(int i = 0 ; i < 10; ++i)
    System.out.println("Configuration Management Second Day"); 
+System.out.println("Configuration Management Second Day1");
```
Comment (Husseinsobieh) [inline src/Main2.java:8]: DRY concept dont repeat yourself

**Review by Omniasalman — APPROVED** [review]

**Review by MaryamMagdy570 — APPROVED** [review]

**Review by BasantMohamed33 — APPROVED** [review]

**Review by Ali-Abdelhakem-00 — APPROVED** [review]

**Review by Salwa1012 — APPROVED** [review]

**Review by Husseinsobieh — APPROVED** [review]

**Review by AbdallahALBanna — APPROVED** [review]

File: src/Main2.java (hunk 1 above)
Comment (AhmedOsama96) [inline src/Main2.java:10]: code duplicate

**Review by AhmedOsama96 — APPROVED** [review]

**Review by AhmedTawfik98 — COMMENTED** [review]
TOO MUCH CHANGES OMAR

**Pull request merged**